controllers/articles.controllers.js

```diff
@@ -10,3 +10,26 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getArticles = (req, res, next) => {
+  let articleCommentCount 
+  selectArticleIdByCount().then((body) => {
+    articleCommentCount = body
+  }).then(() => {
+    selectArticles().then((articles) => {
+      for (let i=0; i<articles.length; i++) {
+        for (let j=0; j<articleCommentCount.length; j++) {
+        delete articles[i].body
+        if (articles[i].article_id === articleCommentCount[j].article_id) {
+          articles[i].comment_count = Number(articleCommentCount[j].count)
+        }
+        if (!articles[i].comment_count) {
+          articles[i].comment_count = 0
+        }
+      }   
```

Comment:
This is a very complicated model! I appreciate all you've done to get this the way it needs to be presented back to the client. I'd like you to have a look into CAST to create a comment count in the query rather than outside of it :)